Sources/PrinceOfVersions/Objective-C Helpers/ResponseModel Wrapers/UpdateData/UpdateInfoObjectiveC.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  UpdateInfoObjectiveC.swift
+//  PrinceOfVersions
+//
+//  Created by Ivana Mršić on 29/05/2020.
+//
+
+import Foundation
+
+@objcMembers
+public class UpdateInfoObject: NSObject {
```

Comment:
Should we hide this for swift?

Reply:
No. Same reason as other ObjC exposed classes.